### PR TITLE
add `plot_distribution`

### LINKED ISSUE #16: Port plotting functions to `python`
Overview: 
- [x] Fig. 2 #19 
- [ ] Fig. 3 seems to work nice with an `xr_bitround` wrapper 
- [x] Fig. SI 1: #59 

--- 
The plotting function gives a great overview about the information content present in each bit of a variable.
The port of the function to python should be straight forward and should be preferred over a further call of to Julia.

### PR DESCRIPTION
**Pull request by aaronspring**
- closes #16 
- add `plot_distribution`
- create `graphics.py` and move `plot_bitinformation`

### CODE REVIEW

File: bitinformation_pipeline/graphics.py [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    if gmin < 0 and gmax > 0:
+        bins_neg = np.geomspace(gmin * f, -close_zero, nbins // 2 + 1, dtype=float)
+        bins_pos = np.geomspace(close_zero, gmax * f, nbins // 2, dtype=float)
+        bins = np.concatenate([bins_neg, bins_pos], axis=-1)
+    else:
+        bins = np.geomspace(gmin / f, gmax * f, nbins + 1, dtype=float)
+
+    H = np.zeros((nvars, nbins))
+    for i, v in enumerate(varnames):
+        d = ds[v].data.flatten()
```
Comment (aaronspring) [inline bitinformation_pipeline/graphics.py:223]: Maybe warn if unnecessary many gridppints provided?

**Review by observingClouds — COMMENTED** [review]
Please check my inline comments about the `seaborn` dependency and the comment about the mentioned video.

File: examples/quick-start.ipynb [hunk 2]
```diff
@@ -5,7 +5,9 @@
    "id": "3f2c9098",
    "metadata": {},
    "source": [
-    "# Example to retrieve bitinformation"
+    "# Example to retrieve bitinformation",
+    "\n",
+    "python wrapper for bitinformation.jl, add paper and video"
```
[the remark this answers is not in the document]
Comment (aaronspring) [reply examples/quick-start.ipynb:10]: Sorry. That's already in the Merged readme PR. I will delete that reference here.

File: environment.yml [hunk 3]
```diff
@@ -6,6 +6,7 @@ dependencies:
   - julia
   - pyjulia
   - matplotlib-base
+  - seaborn
```
[the remark this answers is not in the document]
Comment (aaronspring) [reply environment.yml:9]: True. Any matplotlib cm will be fine.

**Review by observingClouds — APPROVED** [review]

**Pull request merged**